pkgs/data/fonts/dm-sans/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, fetchzip }:
+
+fetchzip rec {
+  pname = "dm-sans";
+  version = "1.002";
+  url = "https://github.com/googlefonts/dm-fonts/releases/download/v${version}/DeepMindSans_v${version}.zip";
```

```suggestion
  url = "https://github.com/googlefonts/dm-fonts/releases/download/v${version}/DeepMindSans_v${version}.zip";
  stripRoot = false;
```
Apparently this is required but it was commented out so I suggested to remove it.

---

thanks! due to an error on my side, at the time, I haven't been able to test it locally.
I will correct it. Lets get this finally merged ! :)